protocol/directory_test.go

```diff
@@ -80,6 +80,13 @@ func TestRegisterExistedUserWithTB(t *testing.T) {
 }
 
 func TestRegisterWithoutTB(t *testing.T) {
+	// workaround for #110
```

Comment:
I mean that since we're panicking early, the rest of the code won't be executed and will likely accumulate bugs or otherwise be unuseful over time.